### ISSUE TITLE
Graph commit transactions issue on 2021.1.2

### ISSUE THREAD

**Issue by aguinaldoabbj** [technige/py2neo#903]
Hello all,

I'm getting this error

```
/opt/conda/lib/python3.9/site-packages/py2neo/data/__init__.py in __db_merge__(self, tx, primary_label, primary_key)
    299         for (pl, pk, labels), nodes in node_dict.items():
    300             if pl is None or pk is None:
--> 301                 raise ValueError("Primary label and primary key are required for MERGE operation")
    302             pq = unwind_merge_nodes_query(map(dict, nodes), (pl, pk), labels)
    303             pq = cypher_join(pq, "RETURN id(_)")

ValueError: Primary label and primary key are required for MERGE operation
```

when I'm trying to commit a subgraph with 2021.1.2. I'm using the very same piece of code that used to work until version 2021.1.1:

```
for p in subgraphs:
    tx = graph.begin()
    tx.merge(p)
    graph.commit(tx)
```
Primary label and key were as always defined when I created a Node Object. Same corde works on 2021.1.1.

**Comment by technige** [comment]
Can you share your `Model` subclass?

**Comment by technige** [comment]
Can you share a typical value for `p`?

**Comment by technige** [comment]
And fixed in release 2021.1.3
https://pypi.org/project/py2neo/2021.1.3/
